Admin/BaseMediaAdmin.php

```diff
@@ -106,7 +106,7 @@ public function getNewInstance()
 
         if ($this->hasRequest()) {
             if ($this->getRequest()->isMethod('POST')) {
-                $media->setProviderName($this->getRequest()->get(sprintf('%s[providerName]', $this->getUniqid()), null, true));
+                $media->setProviderName($this->getRequest()->get(sprintf('%s', $this->getUniqid()), null, true)['providerName']);
```

Comment:
You can maybe try detecting [this method](https://github.com/symfony/http-foundation/compare/2.7...2.8#diff-313a1e2d3e8584eebf87bb466619504eL211) instead, [it should work](https://3v4l.org/JXrBm)